## Requirements

The ```nhtemp``` built-in data set gives the mean annual temperature in New Haven, CT, for the years 1912 to 1971.

Suppose you want to test whether the average over the first 25 years (1912 to 1936) of the data is statistically significantly different than the average over the last 25 years (1947 to 1971) of the data set. You create two vectors from the data set: ```first25``` and ```last25```, using the following code:

```{r}
first25 <- nhtemp[1:25]
last25 <- nhtemp[36:60]
```

Compute a test to see if these two vectors have the same mean. 

What type of *t*-test should you use? 

* Single sample
* Independent 
* Dependent 

### The correct answer is Dependent t-test!


Within a text document, discuss the following: 

* The problem to be solved
* The hypotheses 
* The results of the hypothesis test and the conclusion
>What is the *p*-value for this test? Based on this *p*-value, do you reject the null hypothesis?


---

# Look at Built in Dataset

In [18]:
head(nhtemp)

[1] 49.9 52.3 49.4 51.1 49.4 47.9

# Change to Dataframe

In [19]:
# We need to change the built in dataset called `rivers` into a dataframe to 
# properly look at it and create the structure for the dataset. 
# You can see the difference in structure from the line above and below.  

temp = data.frame(nhtemp)

# Look at Data

In [20]:
head(temp)

nhtemp
1 49.9  
2 52.3  
3 49.4  
4 51.1  
5 49.4  
6 47.9

# Create a Vector:
### We want to know what the first 25 years and the last 25 years have any differences. 

In [21]:
first25 <- nhtemp[1:25]
last25 <- nhtemp[36:60]

# Run Dependent T-test

In [22]:
t_dep <- t.test(first25, last25, paired = TRUE)
t_dep


	Paired t-test

data:  first25 and last25
t = -3.924, df = 24, p-value = 0.0006383
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -2.2645321 -0.7034679
sample estimates:
mean of the differences 
                 -1.484 


# State Hypothesis:

- Null: There is no difference
- Alternative: There is a difference
    - p-value = 0.0006383

Reject Null and Accpet Alternative; There is a difference from the first 25 years to the last 25 years average tempertures in New Haven, CT, for the years 1912 to 1971.


# Graphing Data for an Dependent T-test

In [23]:
library(reshape2)

In [30]:
temp1 <- melt(temp, measure.vars = "nhtemp")